src/classes/RD2_DataMigrationMapper.cls

```diff
@@ -216,19 +216,24 @@ public class RD2_DataMigrationMapper {
      */
     private void setStartDate(npe03__Recurring_Donation__c rd) {
```

Comment:
The method description should be updated to reflect the Date_Established__c changed